### PR TITLE
add: `ferium-bin`

### PR DESCRIPTION
**Pull request by Henryws**
https://github.com/gorilla-devs/ferium/issues/42

### CODE REVIEW

File: packages/ferium-bin/ferium-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+prepare() {
+  true
+}
+
+build() {
+  true
+}
+
+install() {
+  sudo install -Dm755 ferium "${STOWDIR}/${name}/usr/bin/${pkgname}"
```
Comment (wizard-28) [inline packages/ferium-bin/ferium-bin.pacscript:19]: Install the shell completions too

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/ferium-bin/ferium-bin.pacscript (hunk 1 above)
Comment (Henryws) [reply packages/ferium-bin/ferium-bin.pacscript:19]: The binary is the only thing in the archive. Completions are not provided in it.

**Review by wizard-28 — APPROVED** [review]

File: packages/ferium-bin/ferium-bin.pacscript (hunk 1 above)
Comment (theRookieCoder) [reply packages/ferium-bin/ferium-bin.pacscript:19]: Is it not possible to run the command to generate the completions?

Comment (Henryws) [reply packages/ferium-bin/ferium-bin.pacscript:19]: Oh I didn't know ferium had the ability to generate completions. Lemme work on that

Comment (theRookieCoder) [reply packages/ferium-bin/ferium-bin.pacscript:19]:
```text
-> ferium complete --help
ferium-complete 4.1.5
Generate shell auto completions to stdout for the specified shell

USAGE:
    ferium complete <SHELL>

ARGS:
    <SHELL>    The shell to generate auto completions for

OPTIONS:
    -h, --help       Print help information
    -V, --version    Print version information
```

Comment (wizard-28) [reply packages/ferium-bin/ferium-bin.pacscript:19]: The AUR PKGBUILD for reference: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=ferium

Comment (Henryws) [reply packages/ferium-bin/ferium-bin.pacscript:19]: https://github.com/pacstall/pacstall-programs/pull/1371

**Pull request merged**